Dockerfile

```diff
@@ -11,7 +11,8 @@ RUN apt-get update && \
 RUN curl https://sh.rustup.rs -sSf | sh -s -- -y && \
          export PATH=$PATH:$HOME/.cargo/bin && \
          scripts/init.sh && \
-         cargo +nightly-2020-10-06 build --$PROFILE
+         cargo test && \
+         cargo build --$PROFILE
```

Comment:
@huythanhcere 
Move back `+nightly-2020-10-06`. We will do migration separately.

Reply:
After tried with "+nightly-2020-10-06", i tried to remove it. This change is temporary. I will move back in the next commit.